mariari/why-elixir.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Why Anoma
+publish_date:
```

Comment:
- Can add a publish date - can be today / tomorrow. Won't update on the blog until the PR is merged. 
- For image you want to create a media folder then reference the item in image. I'm not sure what you have in mind. Typically, I use one of the AI image generating services to get image that gestures at the article. 
- For excerpt: Anoma is a decentralized operating system with heterogeneous
trust. For developing Anoma we've been using [Elixir](https://elixir-lang.org/). This choice is somewhat unconventional, and thus begs the question "why write Anoma in Elixir!?"
- For Category the choices are: compilers, distributed-systems, ecosystem, community, cryptography, applications, security, community